app/code/Magento/Sales/view/adminhtml/templates/order/address/form.phtml

```diff
@@ -10,12 +10,12 @@
     </div>
 </div>
 
-<fieldset class="fieldset admin__fieldset-wrapper">
+<fieldset class="fieldset admin__fieldset-wrapper admin__fieldset">
```

Comment:
Looks like these changes are for https://github.com/magento/magento2/pull/23482 can you please remove them from the pull request and leave only code changes related to this particular PR

Reply:
Hi @sivaschenko ,